sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -62,14 +62,45 @@ public async Task StartBuildModel(bool singlePage)
             var trainingFilesUri = new Uri(singlePage ? TestEnvironment.BlobContainerSasUrl : TestEnvironment.MultipageBlobContainerSasUrl);
             var modelId = Recording.GenerateId();
 
-            BuildModelOperation operation = await client.StartBuildModelAsync(trainingFilesUri, modelId);
+            BuildModelOperation operation = await client.StartBuildModelAsync(trainingFilesUri, DocumentBuildMode.Template, modelId);
             await operation.WaitForCompletionAsync();
 
             Assert.IsTrue(operation.HasValue);
 
             DocumentModel model = operation.Value;
 
             ValidateDocumentModel(model);
+
+            Assert.AreEqual(1, model.DocTypes.Count);
+            Assert.IsTrue(model.DocTypes.ContainsKey(modelId));
+
+            DocTypeInfo docType = model.DocTypes[modelId];
+
+            Assert.AreEqual(DocumentBuildMode.Template, docType.BuildMode);
```

Comment:
Notice we didn't have any kind of doctype validation in these tests, so added a similar piece of code to the Build, Copy, and Compose tests.